### DISCUSSION TITLE
包索引更新

### DISCUSSION THREAD

**Discussion by PeakRacing** [xmake-io/xmake#3480] [Q&A] [answered]
你好，如果[xmake-repo](https://github.com/xmake-io/xmake-repo)仓库有更新，我需要如何更新本都包索引呢？我看xmake require --upgrade是把包也更新了，但是我只想更新索引需要执行什么命令？

**Comment by waruqi** [comment, marked as answer]
xrepo update-repo

**Comment by PeakRacing** [comment]
收到，感谢
